packages/commonwealth/client/scripts/views/modals/archive_thread_modal.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import 'modals/archive_thread_modal.scss';
+import type Thread from '../../models/Thread';
+import app from 'state';
+import { CWButton } from '../components/component_kit/cw_button';
+import { CWText } from '../components/component_kit/cw_text';
+import { useEditThreadMutation } from 'state/api/threads';
+
+import {
+  notifyError,
+  notifySuccess,
+} from '../../controllers/app/notifications';
+
+type ArchiveThreadModalProps = {
+  onModalClose: () => void;
+  thread: Thread;
+};
+
+export const ArchiveThreadModal = ({
```

Comment:
change name of the file `archive_thread_modal.tsx` => `ArchiveThreadModal.tsx`

Reply:
Done!